src/selenium/driver.cr

```diff
@@ -33,7 +33,11 @@ abstract class Selenium::Driver
     @service.try &.start
     base_url ||= @service.not_nil!.base_url
     @http_client = HttpClient.new(base_url: base_url)
-    @command_handler = CommandHandler.new(@http_client)
+    @command_handler = command_handler
+  end
+
+  private def command_handler : CommandHandler
```

Comment:
This would override the getter on line 29. I don't think this would be necessary especially since the `getter` would also provide a little memoization unless I'm missing something here?

Reply:
I don't recall why I made this change :-) . I will revert it.